src/ingestors/zora-mint/index.ts

```diff
@@ -0,0 +1,102 @@
+import { MintContractOptions, MintIngestor, MintIngestorResources } from '../../lib/types/mint-ingestor';
+import { MintIngestionErrorName, MintIngestorError } from '../../lib/types/mint-ingestor-error';
+import { MintInstructionType, MintTemplate } from '../../lib/types/mint-template';
+import { MintTemplateBuilder } from '../../lib/builder/mint-template-builder';
+import { zoraMintAbi } from './abi';
+import {  getZoraMintPriceInEth,getZoraContractMetadata } from './onchain-zora';
+import { urlForValidZoraPage,zoraOnchainIdDataFromUrl } from './offchain-metadata';
+// import { getProhibitionContractMetadata} from './onchain-zora';
+// import { Address, encodeAbiParameters } from "viem";
+// 
+export class zoraIngestor implements MintIngestor {
```

Comment:
It is idiomatic in Typescript to begin with capital letters 🙏 

This is also a good reminder that we should add a lint check to submission